.storybook/3.COLORS.stories.mdx

```diff
@@ -75,49 +76,49 @@ var(--color-border-muted)
 
 /** Overlays */
 var(--color-overlay-default)
-var(--color-overlay-inverse)
+var(--color-overlay-inverse) [DEPRECATED]
 
 /** User Actions */
 var(--color-primary-default)
 var(--color-primary-alternative)
 var(--color-primary-muted)
 var(--color-primary-inverse)
-var(--color-primary-disabled)
+var(--color-primary-disabled) [DEPRECATED]
 
-var(--color-secondary-default)
-var(--color-secondary-alternative)
-var(--color-secondary-muted)
-var(--color-secondary-inverse)
-var(--color-secondary-disabled)
+var(--color-secondary-default) [DEPRECATED]
+var(--color-secondary-alternative) [DEPRECATED]
+var(--color-secondary-muted) [DEPRECATED]
+var(--color-secondary-inverse) [DEPRECATED]
+var(--color-secondary-disabled) [DEPRECATED]
 
 /** States */
 /** Error */
 var(--color-error-default)
 var(--color-error-alternative)
 var(--color-error-muted)
 var(--color-error-inverse)
-var(--color-error-disabled)
+var(--color-error-disabled) [DEPRECATED]
 
 /** Warning */
 var(--color-warning-default)
-var(--color-warning-alternative)
+var(--color-warning-alternative) [DEPRECATED]
 var(--color-warning-muted)
 var(--color-warning-inverse)
-var(--color-warning-disabled)
+var(--color-warning-disabled) [DEPRECATED]
 
 /** Success */
 var(--color-success-default)
-var(--color-success-alternative)
+var(--color-success-alternative) [DEPRECATED]
 var(--color-success-muted)
 var(--color-success-inverse)
-var(--color-success-disabled)
+var(--color-success-disabled) [DEPRECATED]
 
 /** Info */
 var(--color-info-default)
-var(--color-info-alternative)
+var(--color-info-alternative) [DEPRECATED]
 var(--color-info-muted)
 var(--color-info-inverse)
-var(--color-info-disabled)
+var(--color-info-disabled) [DEPRECATED]
 ```
 
 ### **Component colors** (tier 3)
```

Comment:
Adding deprecated tags to color docs. Will remove update and remove instances in a separate PR.

<img width="1440" alt="Screen Shot 2022-05-17 at 2 28 48 PM" src="https://user-images.githubusercontent.com/8112138/168913258-049f4fe7-de2e-43c5-9aa8-3d453f703495.png">